### ISSUE TITLE
Problem with loopHorizontal:false and  jquery.fullpage.extensions.min.js

### ISSUE THREAD

**Issue by spaszwerg** [alvarotrigo/fullPage.js#2548]
Script seems to crash when using the extension file. JS-Console alerts: TypeError: t is undefined

Hi,
i´m creating a fullPage website for a customer. The site is intended to work with the extensions dragandmove and resetSlides in the end. So I included the extension library from the dist-folder instead of the normal one. 

My js in the html-file looks like this:
```html
<script src="js/jquery.min.js"></script> (v3.1.1.)
<script src="js/jquery.fullpage.extensions.min.js"></script>
<script>
            $(document).ready(function() {
                $('#fullpage').fullpage({
                    loopHorizontal: false,
                    navigation: true,
                    verticalCentered: false,
                    navigationTooltips: ['view1', 'view2', 'view3', 'view4', 'view5', 'view6'],
                    menu: '.mobileNavDropdown',
                    anchors:['view1', 'view2', 'view3', 'view4', 'view5', 'view6'],
                    normalScrollElements:'.noScroll'
                });
            });
        </script> 
```
### Steps to reproduce it
fullPage is working as expected, but if I´m on the last slide of an horizontal row and swipe right/right arrow on keyboard, the script crashes giving a console error: "TypeError: t is undefined".
I thought that might be because I didn´t include the extensions in the beginning. 
After my customer got these from your shop I included them, but no change. The script still stops working if I try to slide to the next slide, that doesn´t exist.

The problem disapperas when setting the loopHorizontal option to true, but thats not what the customer wants.

### Versions 
Tested on Win10 with FF51 and Chrome 56 as well as several Android smartphones.

Any hints or advices are welcome.

**Comment by alvarotrigo** [comment]
> fullPage is working as expected, but if I´m on the last slide of an horizontal row and swipe right/right arrow on keyboard, the script crashes giving a console error: "TypeError: t is undefined".

Are you saying you were having this error before using the extensions?
If so, can you please provide an isolated reproduction in jsfiddle or codepen with no external CSS and JS?

**Comment by spaszwerg** [comment]
Thank you for the quick reply.
I created a fiddle here:
https://jsfiddle.net/schuetzmedia/ekr890vd/7/

**Comment by alvarotrigo** [comment]
Thanks for it.
Solved [in the dev branch](https://github.com/alvarotrigo/fullPage.js/blob/dev/dist/jquery.fullpage.extensions.min.js) and will get merged in the next release.

**Comment by spaszwerg** [comment]
Hi, thats good to hear.
When will the next release be?
My deadline is close ;o)

**Comment by alvarotrigo** [comment]
> When will the next release be?

Probably this week. But you can use [the file in the dev version](https://github.com/alvarotrigo/fullPage.js/blob/dev/dist/jquery.fullpage.extensions.min.js). It will solve your issue.

**Comment by spaszwerg** [comment]
Perfect! Thanks a lot.

**Comment by alvarotrigo** [comment]
Solved in the new release 2.9.3
